modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaResteasyServerCodegen.java

```diff
@@ -9,10 +10,11 @@
 import java.io.File;
 import java.util.*;
 
-public class JavaResteasyServerCodegen extends AbstractJavaJAXRSServerCodegen implements JbossFeature {
+public class JavaResteasyServerCodegen extends AbstractJavaJAXRSServerCodegen implements JbossFeature, BeanValidationFeatures {
 
+	protected boolean useBeanValidation = true;
```

Comment:
Please use 4-space instead of tab.

What about setting `useBeanValidation` to false to start with similar to what we've done for Java API client?